Signal/translations/en.lproj/Localizable.strings

```diff
@@ -718,6 +718,13 @@
 /* Navbar title */
 "SETTINGS_ABOUT" = "About";
 
+/* This setting is used to switch between new-style WebRTC calling and old-style "RedPhone" calling. */
+"SETTINGS_ADVANCED_WEBRTC" = "Enable WebRTC Calling";
+
+/* This setting is used to switch between new-style WebRTC calling and old-style "RedPhone" calling. */
+"SETTINGS_ADVANCED_WEBRTC_FAILED_TITLE" = "Error";
+"SETTINGS_ADVANCED_WEBRTC_FAILED_MESSAGE" = "Could not update your preferences.";
```

Comment:
Can you clarify?  I don't understand the workflow.  So, I: 

1. add/change usage of NSLocalizedString() as I have it here.
2. run the script.
3. find the entries added to Localizable.strings and insert the correct copy there.

If that's right, isn't that just like what I've done here, except the ordering of entries in Localizable.strings?